### ISSUE TITLE
[Issue]: Apply new card container design

### ISSUE THREAD

**Issue by ClaireValdivia** [usdigitalresponse/usdr-gost#3154]
### Why is this issue important?

We want to visually distinguish the different parts of the grant details page; the goal is for the page organization to be more clear to users, especially as we plan to roll out more collaboration features on the page. We plan to use this design in the future as we design an improved dashboard, public profile, and my profile pages.  

### Current State

The grant details page has a solid white background 

### Expected State

Apply the new card container from the design library to the grant details page to show contrast between the page background and cards on the page - [Design Library - Card Container](https://www.figma.com/design/HWu4iIjcoX2txuN543qzIB/USDR-Design-System?node-id=2633-138845&t=M5gdDZqa1qoHNSxs-0)

### Implementation Plan

The following functions need to be added...

**Comment by jeffsmohan** [comment]
@ClaireValdivia I wrote up implementation details here per Ty's request, and I noticed this ticket is largely blocked by the rest of the updates to the Grant Details page design. (For example, until the link/print buttons are moved into the main section, it's unclear how they would fit into a card-based layout.) Can you make sure that dependency is properly captured, since I'm not sure where all the other tickets are? Thanks!

**Comment by ClaireValdivia** [comment]
Thanks @jeffsmohan ! I'll update the issue to clarify as well, but this should not be dependent on other issues as the card design should be applied to following:
- the content in the left column (as shown in the designs)
- the Assign Grant Section
- the Team status section

Right now, we don't need to apply the styling to the apply on grants.gov, print, and copy buttons, as they will become part of the left column container. For now they can remain on the page as they are, with the three other sections showing as cards. Let me know if that makes sense!

**Comment by ClaireValdivia** [comment]
yeah appreciate the question as we hadn't done any specific designs for that. 

and confirming that it does not need a feature flag and can go live as soon as it's merged!

**Comment by masimons** [comment]
Dropping this ticket to focus on forecasted grants per discussion in weekly grants meeting

**Comment by ClaireValdivia** [comment]
This looks good, thanks @mayabose! I know that @caitlinwinner has a few adjustment requests, so none blocking deployment so we'll make a new issue for adjustments.
